packages/client/bin/cli.ts

```diff
@@ -199,8 +206,68 @@ async function run() {
   // give network id precedence over network name
   const chain = args.networkId ?? args.network ?? 'mainnet'
 
-  // configure common based on args given
+  // configure accounts for mining and prefunding in a local devnet
+  const accounts: [Address, Buffer][] = [] // [address, privateKey]
```

Comment:
(my code again) we can inline this comment now that we are on latest TS :)

```suggestion
  const accounts: [address: Address, privateKey: Buffer][] = []
```